dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -0,0 +1,364 @@
+            }
+
+            ret.add(i);
+        }
+        return ret;
+    }
+
+    public static List<Folder> convertDotConnectMapToFolder(List<Map<String,String>> results)
+            throws ParseException {
+
+        Folder folder;
+        List<Folder> ret;
+
+        ret = new ArrayList<>();
+
+        if(results != null && !results.isEmpty()){
+            for (Map<String, String> map : results) {
+                folder = new Folder();
+                folder.setInode(map.get("inode"));
+                folder.setName(map.get("name"));
+                folder.setTitle(map.get("title"));
+
+                if (map.get("show_on_menu") != null && !map.get("show_on_menu").isEmpty()) {
+                    folder.setShowOnMenu(Boolean.parseBoolean(map.get("show_on_menu")));
+                }
+
+                if (map.get("sort_order") != null && !map.get("sort_order").isEmpty()){
+                    folder.setSortOrder(Integer.parseInt(map.get("sort_order")));
+                }
+
+                folder.setFilesMasks(map.get("files_masks"));
+
+                folder.setIdentifier(map.get("identifier"));
+
+                folder.setDefaultFileType(map.get("default_file_type"));
+
+                if (map.get("mod_date") != null && !map.get("mod_date").isEmpty()){
+                    folder.setModDate(df.parse(map.get("mod_date")));
+                }
+
+                ret.add(folder);
+            }
+        }
+        return ret;
+    }
+
+    public static List<Container> convertDotConnectMapToContainer(List<Map<String,String>> results)
+            throws ParseException {
+        Container container;
+        List<Container> ret;
+
+        ret = new ArrayList<>();
+
+        if(results != null && !results.isEmpty()){
+            for (Map<String, String> map : results) {
+                container = new Container();
+                container.setInode(map.get("inode"));
+                container.setCode(map.get("code"));
+                container.setPreLoop(map.get("pre_loop"));
+                container.setPostLoop(map.get("post_loop"));
+                if (map.get("show_on_menu") != null && !map.get("show_on_menu").isEmpty()) {
+                    container.setShowOnMenu(Boolean.parseBoolean(map.get("show_on_menu")));
+                }
+
+                container.setTitle(map.get("title"));
+
+                if (map.get("mod_date") != null && !map.get("mod_date").isEmpty()){
+                    container.setModDate(df.parse(map.get("mod_date")));
+                }
+
+                container.setModUser(map.get("mod_user"));
+
+                if (map.get("sort_order") != null && !map.get("sort_order").isEmpty()){
+                    container.setSortOrder(Integer.parseInt(map.get("sort_order")));
+                }
+
+                container.setFriendlyName(map.get("friendly_name"));
+
+                if (map.get("max_contentlets") != null && !map.get("max_contentlets").isEmpty()){
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "max_contentlets" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)